### PR TITLE
fix repeat string field proc

### PR DESCRIPTION
**Pull request by nlgwcy**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes https://github.com/kmesh-net/kmesh/issues/1064

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c [hunk 1]
```diff
@@ -385,6 +386,25 @@ static int get_string_or_msg_field_size(struct op_context *ctx, const ProtobufCF
...
+}
+
+static int get_field_size(struct op_context *ctx, const ProtobufCFieldDescriptor *field)
+{
+    char *value = NULL;
+
+    if (field->type == PROTOBUF_C_TYPE_MESSAGE)
+        return get_msg_field_size(field);
+
+    if (field->type == PROTOBUF_C_TYPE_STRING) {
```
Comment (LiZhenCheng9527) [inline bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c:401]: The case `field->type == PROTOBUF_C_TYPE_STRING` is already excluded in line 519, so do we still need to do this case on this side?

**Comment by hzxuzhonghu** [comment]
@bitcoffeeiux

**Comment by LiZhenCheng9527** [comment]
@nlgwcy This PR needs to be updated

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**